galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/ras/RasSearchCriteriaStatus.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
```

Comment:
Do we need a unit test for this logic ? It might be good.

Reply:
Why the thumbs down ? What does that mean ?
It's not a universal sign that everyone understands I think...

Reply:
Sorry, I did the unit tests, I'm not sure what I was thinking at the time